### PR TITLE
Allow users without `add_todoitem` permission to view an event

### LINKED ISSUE #754: Permission `add_todoitem` required to view event
People without ability of adding TODO items should be able to view the events.

### PR DESCRIPTION
**Pull request by narayanaditya95**
This fixes #754.

@pbanaszkiewicz Please review.

### CODE REVIEW

File: workshops/views.py [hunk 1]
```diff
@@ -937,7 +937,6 @@ def all_events(request):
 
 
 @login_required
-@permission_required('workshops.add_todoitem', raise_exception=True)
```
Comment (narayanaditya95) [inline workshops/views.py]: I think, POST requests that are intentionally generated to simulate form submissions would now end up creating TODOs.
Is that right?

Comment (pbanaszkiewicz) [inline workshops/views.py]: Yes. It's important to only process the `SimpleTodoForm` when user has `workshops.add_todoitem` permission.

Comment (narayanaditya95) [inline workshops/views.py]: `if request.method == "POST" and request.user.has_perm('workshops.add_todoitem'):`
Is this fine (this returns the event details page) or should we return HTTP403 when the user doesn't have the permission?

Comment (pbanaszkiewicz) [inline workshops/views.py]: This is good for now (but please mark with a comment why we check perms in the middle of the view). If we have more forms in that view in future we'll change that conditional.

**Comment by narayanaditya95** [comment]
@pbanaszkiewicz Ready to be merged.

**Comment by pbanaszkiewicz** [comment]
Thanks, @narayanaditya95!

**Pull request merged**